docs/src/components/HeaderWithLink.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import styled from "styled-components";
+import { ButtonLink, Heading, Stack } from "@kiwicom/orbit-components";
+import { Link as LinkIcon } from "@kiwicom/orbit-components/icons";
+
+import { sluggify } from "../utils/common";
+
+const StyledAnchorWrapper = styled.div`
+  margin-top: -10px;
+
+  svg {
+    visibility: hidden;
+  }
+
+  &:hover svg {
+    visibility: visible;
+  }
+`;
+
+interface Props {
+  children: string;
```

Comment:
do not call this props `children` if it's supposed to be string only, call it `href` for instance, because `children` usually means `React.ReactNode` that can be confusing :)